dbhydra/dbhydra_core.py

```diff
@@ -1744,34 +1803,37 @@ def close_connection(self):
 
     def create_database(self):
         try:
-            os.mkdir(self.name)
-            print("Database created")
+            os.mkdir(self.db_directory_path)
+            print("Database directory created")
         except FileExistsError:
-            print("Database is already created")
+            print("Database directory already exists")
 
 
 class XlsxTable(AbstractTable):
-    def __init__(self, db1, name, columns=None, types=None, id_column_name = "id"):
+    def __init__(self, db1: XlsxDB, name, columns=None, types=None, id_column_name = "id"):
         super().__init__(db1, name, columns)
         self.types = types
         self.id_column_name=id_column_name
-        
-        
+        self.db_path: Path = self.db1.db_directory_path / f"{self.name}.xlsx"
+
     def create(self):
-        df=pd.DataFrame(columns=self.columns)
-        df.to_excel(self.db1.name + "\\" + self.name + ".xlsx",index=False)
-    
+        if not self.db_path.exists():
+            df=pd.DataFrame(columns=self.columns)
+            df.to_excel(self.db_path, index=False)
+        else:
+            print(f"Table '{self.name}' already exists")
+
     def drop(self):
         pass
 
     def select_to_df(self):
         try:
-            df = pd.read_excel(self.db1.name + "//" + self.name + ".xlsx")
+            df = pd.read_excel(self.db_path)
             # cols=df.columns
             # print(cols)
             # df.set_index(cols[0],inplace=True)
             # df.drop(df.columns[0],axis=1,inplace=True)
-
+            df.replace({np.nan: None}, inplace=True)
```

Comment:
I am not sure here

Reply:
I had issues proper datatype casting using `NULL` values written as a string in excel files. On the other hand, keeping `None` values mapped nicely to empty cell (in excel), and is actually distinguishable from a `NULL` string.

So far in my previous tests it worked well.